synth.metadata

```diff
@@ -28,301 +28,5 @@
         "generator": "gapic-generator-typescript"
       }
     }
-  ],
-  "newFiles": [
```

Comment:
Unrelated, but I don't understand how those file lists work :)

Reply:
You and me both